MagicEightBall/src/components/Form.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from "react";
+import Result from "./Result";
+import Error from "./Error";
+
+const Form = () => {
+  const [question, setQuestion] = useState("");
+  const [isValid, setIsValid] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const handleQuestion = (e) => {
+    setQuestion(e.target.value);
+  };
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (
+      question.match(/^[^?]/) && question.match(/\?$/) //fix regex double or more question mark
+        ? setIsValid(true)
```

Comment:
you can use a single state to check validity. Looks like it's valid when there's no error so you can use only one variable for it and remove second if condition